cmd/genschema/main.go

```diff
@@ -71,7 +81,7 @@ func main() {
 		},
 	}
 
-	schema := r.Reflect(&redpanda.Values{})
+	schema := r.Reflect(&operator.Values{})
```

Comment:
Could we instead just parameterize this script? Running a switch on `os.Args[1]` should be pretty straight forward.